src/pages/office/office-behaviour-map.cat-be.ts

```diff
@@ -0,0 +1,648 @@
+import { OutcomeBehaviourMapping } from '../../providers/outcome-behaviour-map/outcome-behaviour-map.model';
+
+export const behaviourMap: OutcomeBehaviourMapping = {
```

Comment:
Out of interest, is this mapping documented anywhere? In confluence / Jira?

Reply:
There was a spreadsheet for cat b, think was against original ticket in jira. 
No Cat BE spreadsheet, but we have copied from Cat B and used same logic on VehicleChecks as used on tellMe question.

Reply:
Ok just as long as we're sure this displaying logic is uniform across categories